msteams-platform/graph-api/meeting-transcripts/get-transcripts-api.md

```diff
@@ -0,0 +1,883 @@
+                },
+                "calendar@odata.associationLink": "https://graph.microsoft.com/beta/users('14b779ae-cb64-47e7-a512-52fd50a4154d')/calendars('AAMkAGE3NjJhOTVhLTNkZDQtNDE2OS05ZjU0LTJmOGQ0YTY2YTdiZQAuAAAAAAD3AG5jNnlgQJvdCL_KgXJIAQBsww5BlIxtT7iFyYWrXV3AAAAAAAENAAA=')/$ref",
+                "calendar@odata.navigationLink": "https://graph.microsoft.com/beta/users('14b779ae-cb64-47e7-a512-52fd50a4154d')/calendars('AAMkAGE3NjJhOTVhLTNkZDQtNDE2OS05ZjU0LTJmOGQ0YTY2YTdiZQAuAAAAAAD3AG5jNnlgQJvdCL_KgXJIAQBsww5BlIxtT7iFyYWrXV3AAAAAAAENAAA=')"
+                }
+                ```
+
+              </details>
+
+3. **Get meeting ID**: Now, your app can use `joinWebUrl` to get the meeting ID.
+
+    Use the following example to request the online meeting ID:
+
+    ``` http
+    GET https://graph.microsoft.com/beta/users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings?$filter=JoinWebUrl%20eq%20'https://teams.microsoft.com/l/meetup-join/19%3ameeting_MTM5OTY3MGUtNmY4Mi00Yjg4LTk2MDUtY2IyZGRlNmU1ZjA2%40thread.v2/0?context=%7b%22Tid%22%3a%222432b57b-0abd-43db-aa7b-16eadd115d34%22%2c%22Oid%22%3a%2214b779ae-cb64-47e7-a512-52fd50a4154d%22%7d'
+    ```
+
+    The response payload contains the meeting ID in the `id` member of the `value` property.
+    <br>
+    <details>
+    <summary><b>Example</b>: Response payload with meeting ID</summary>
+
+    ```json
+    {
+        "@odata.context": "https://graph.microsoft.com/beta/$metadata#users('14b779ae-cb64-47e7-a512-52fd50a4154d')/onlineMeetings",
+        "value": [
+            {
+                "id": "MSoxNGI3NzlhZS1jYjY0LTQ3ZTctYTUxMi01MmZkNTBhNDE1NGQqMCoqMTk6bWVldGluZ19NVE01T1RZM01HVXRObVk0TWkwMFlqZzRMVGsyTURVdFkySXlaR1JsTm1VMVpqQTJAdGhyZWFkLnYy",
+                "creationDateTime": "2022-04-26T07:41:17.3736455Z",
+                "startDateTime": "2022-04-26T10:30:00Z",
+                "endDateTime": "2022-04-26T11:00:00Z",
+                "joinUrl": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_MTM5OTY3MGUtNmY4Mi00Yjg4LTk2MDUtY2IyZGRlNmU1ZjA2%40thread.v2/0?context=%7b%22Tid%22%3a%222432b57b-0abd-43db-aa7b-16eadd115d34%22%2c%22Oid%22%3a%2214b779ae-cb64-47e7-a512-52fd50a4154d%22%7d",
+                "joinWebUrl": "https://teams.microsoft.com/l/meetup-join/19%3ameeting_MTM5OTY3MGUtNmY4Mi00Yjg4LTk2MDUtY2IyZGRlNmU1ZjA2%40thread.v2/0?context=%7b%22Tid%22%3a%222432b57b-0abd-43db-aa7b-16eadd115d34%22%2c%22Oid%22%3a%2214b779ae-cb64-47e7-a512-52fd50a4154d%22%7d",
+                "chatInfo": {
+                    "threadId": "19:meeting_MTM5OTY3MGUtNmY4Mi00Yjg4LTk2MDUtY2IyZGRlNmU1ZjA2@thread.v2",
+                    "messageId": "0",
+                    "replyChainMessageId": null
+                }
+            }
+        ]
+    }
+    ```
+
+    </details>
+
+    > [!NOTE]
+    > The response payload also contains the `threadID` member in the `chatInfo` property. Your app can use it for obtaining meeting ID if it's subscribed to receive user-level notifications.
```

Comment:
@mankadnandan what does the last sentence mean? I'm a bit confused. The response payload right above already has the meeting ID. Why would the app use the threadId to obtain meeting ID after that?

Reply:
cc. @v-shalinirobinson

Reply:
@v-shalinirobinson  We don't need this note. Please remove the same.

Reply:
The comment is removed following discussion.